paddlenlp/transformers/bert/tokenizer.py

```diff
@@ -362,24 +412,56 @@ def _tokenize(self, text):
     def tokenize(self, text):
         """
         End-to-end tokenization for BERT models.
+
         Args:
             text (str): The text to be tokenized.
         
         Returns:
             list: A list of string representing converted tokens.
+
+        Examples:
+            .. code-block::
+
+                from paddlenlp.transformers import BertTokenizer
+
+                berttokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+                tokens = berttokenizer.tokenize('He was a puppeteer')
+                '''
+                ['he', 'was', 'a', 'puppet', '##eer']
+                '''
+
         """
+
         return self._tokenize(text)
 
     def convert_tokens_to_string(self, tokens):
         """
         Converts a sequence of tokens (list of string) in a single string. Since
```

Comment:
in a single string -> to a single string

Reply:
done